rainflame/geopolygonize
# Save tile creation progress

This PR modifies `GeoPolygonizer` to report where it saves tiles so if it breaks, the user can re-input the directory via the `--tile-dir` option and continue where they left off to finish making the final output shapefile.

**Test**

1. Run `./tests/run_e2e_test.sh sisters`.
2. Get tile directory.
3. Observe CLI finish making all the tiles and output final shapefile.
4. Rerun `./tests/run_e2e_test.sh sisters`, this time with the `--tile-dir` option.
5. Observe CLI speed through tile creation and just work on stitching to remake final shapefile.
